examples/complete/variables.tf

```diff
@@ -71,3 +71,9 @@ variable "enable_sqs_events_on_access_log_access" {
   type        = bool
   default     = false
 }
+
+variable "private_ip" {
+  description = "The private IP address to assign to the bastion"
+  type        = string
+  default     = ""
```

Comment:
should this be the same default value as the variables.tf in the module?

Reply:
> left one comment on the default var values
> 
> did you happen to test both providing an input and leaving the defaults?

@ntwkninja I did, for testing I left the default blank and also entered an IP. As for the top level var being `""` as apposed to the variable in the module being `null` that was really a personal choice, I felt that by using `""` it was a bit easier for someone to understand they can just insert what they want. Happy to change to null if that makes more sense to others.

Reply:
My 2 cents, I prefer the use of `null` instead of `""`. @JaseKoonce I am content, but I'll let @ntwkninja or @brianrexrode decide.

Reply:
No preference here and no issue approving this PR or a future one that changes the example to null.